### PR TITLE
284 add contributor role

### PR DESCRIPTION
**Pull request by mnuzzose**

### CODE REVIEW

File: backend/database/models/organization.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+from backend.database.models._assoc_tables import organization_user
+
+
+class Organization(db.Model, CrudMixin):
+    id = db.Column(db.String, primary_key=True)
+    name = db.Column(db.Text)
+    url = db.Column(db.Text)
+    contact_email = db.Column(db.Text)
+    reported_incidents = db.relationship(
+        'Incident', backref='organization', lazy="select")
```
Comment (DMalone87) [inline backend/database/models/organization.py:11]: The backref here should still be source.

File: backend/database/models/incident.py
Comment (DMalone87) [inline backend/database/models/incident.py:1]: The relationship of an organization to an incident is still going to be that of a source -> Incident. The backref on org and the labelling here should stay mostly the same.

File: backend/database/models/incident.py [hunk 2]
```diff
@@ -59,10 +59,10 @@ class Incident(db.Model, CrudMixin):
     """The incident table is the fact table."""
 
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
-    source_id = db.Column(
```
Comment (DMalone87) [inline backend/database/models/incident.py:62]: These should be reverted.

File: backend/database/models/incident.py [hunk 3]
```diff
@@ -131,8 +131,8 @@ def __repr__(self):
     # case_id = db.Column(db.Integer)  # TODO: foreign key of some sort?
 
 
-class SourceDetails(db.Model):
-    id = db.Column(db.Integer, primary_key=True)  # source details id
+class OrganizationDetails(db.Model):
```
Comment (DMalone87) [inline backend/database/models/incident.py:134]: This is still going to be SourceDetails.

**Review by DMalone87 — CHANGES_REQUESTED** [review]

File: backend/database/models/organization.py (hunk 1 above)
Comment (mnuzzose) [reply backend/database/models/organization.py:11]: If we're renaming `Source` to `Organization` on the back end for consistency with the front end, the database should also reflect that for consistency.

**Comment by DMalone87** [comment]
Gonna go ahead and merge this and make the changes after.

**Pull request merged**